src/components/CampaignForm.tsx

```diff
@@ -86,9 +152,10 @@ const CreateCampaignForm = ({ onSubmit, onCancel }: CreateCampaignFormProps): JS
     ];
 
     const handleSubmit = (values: Record<string, string>) => {
-        const { info_text, status, title, visibility } = values;
+        const { info_text, status, title, visibility, manager_id } = values;
         onSubmit({
             ...info_text ? { info_text } : null,
+            ...manager_id ? { manager_id } : null,
```

Comment:
There is a bug here (or somewhere else related to this), which means you can't remove the campaign manager from a campaign once set. The `null` is not being included in the `PATCH`.

Reply:
When I try to include null in patch, I get an error Invalid parameter, NONE should be an instance of int. 
![Screenshot 2021-07-07 at 14 39 15](https://user-images.githubusercontent.com/62522290/124760509-1bef6f00-df31-11eb-838b-58c15d60d1c9.png)
That's the patch schema, it won't accept null. Is the way to remove the manager a PUT?

Reply:
That sounds like a bug in Core. Please file an issue and add to R28. :)

Reply:
done